src/Core/Content/Flow/FlowException.php

```diff
@@ -52,4 +56,26 @@ public static function transactionFailed(\Throwable $previous): self
             ),
         };
     }
+
+    /*
+    * @deprecated tag:v6.8.0 - StateMachineException::stateMachineNotFound will be replaced with FlowException::stateMachineNotFound
+    */
+    public static function stateMachineNotFound(string $stateMachineName): self|StateMachineException
+    {
+        if (!Feature::isActive('v6.8.0.0')) {
+            Feature::triggerDeprecationOrThrow(
```

Comment:
this will always trigger a deprecation notice which could not be fixed until 6.8

you can add a `reason:blabla` to the deprecated annotation, like we do on other places. with this, you do not need to add a call of `Feature::triggerDeprecationOrThrow`